frontends/web/src/components/bitboxbase/connectbase.tsx

```diff
@@ -83,24 +83,18 @@ export class ConnectedBase extends Component<Props, State> {
         if (!blockInfo) {
             return null;
         }
-        let blocks = '';
-        let difficulty = '';
-        let alias = '';
-        let i = '';
-        const blockInfoArr = blockInfo.split(' ');
-        blocks = blockInfoArr[0];
-        difficulty = blockInfoArr[1];
-        i = blockInfoArr[2];
-        alias = blockInfoArr[3];
+
+        const blocks = 'blocks';
+        const difficulty = 'difficulty';
+        const lightningAlias = 'lightningAlias';
 
         return (
                 <div class="row">
                     <div class="flex flex-1 flex-row flex-between flex-items-center spaced">
-                        <p>Block Number: {blocks}</p>
-                        <p>Difficulty: {difficulty}</p>
-                        <p>Incrementer: {i}</p>
+                        <p>Block Number: {blockInfo[blocks]}</p>
+                        <p>Difficulty: {blockInfo[difficulty]}</p>
                         <p>Device ID: {bitboxBaseID}</p>
-                        <p>Lightning Alias: {alias}</p>
+                        <p>Lightning Alias: {blockInfo[lightningAlias]}</p>
```

Comment:
since `const lightningAlias = 'lightningAlias'` is always lightningAlias, you can just `{blockInfo.lightningAlias}` and remove `const lightningAlias`. Same for `blockInfo.blocks` and `blockInfo. difficulty `

Reply:
Typescript does not allow object referencing without giving the object a type first.I took this lazy way out, but should do it properly.